data/seeds/013-role_tickets.js

```diff
@@ -0,0 +1,28 @@
+const dummyData = [
+  {
+    submitted_by: '00ulthapbErVUwVJy4x6',
+    subject_id: '00ulthapbErVUwVJy4x6',
+    requested_role: 1,
+    approved_by: '00ulthapbErVUwVJy4x6',
+    comments: 'This is my 1st dummy data comment',
+    pending: true,
+    resolved: false,
+  },
+  {
+    submitted_by: '00ulthapbErVUwVJy4x6',
+    subject_id: '00ulthapbErVUwVJy4x6',
+    requested_role: 1,
+    approved_by: '00ulthapbErVUwVJy4x6',
+    comments: 'This is my 2nd dummy data comment',
+    pending: true,
+    resolved: false,
+  },
+];
```

Comment:
I would love to see this tested with a little more dummy data with a bit more variety in the future, to give our Front-End team more data to work with rendering, but this is looking good so far!